data/Templates/Hl7v2/DataType/_EIPIdentifier.liquid

```diff
@@ -1,26 +1,24 @@
 {% if EIP -%}
-
-"value":"{{ EIP.1.1 }}",
-"value":"{{ EIP.2.1 }}",
-"type":
-{
-    "coding":
-    [
-        {
-            {% if EIP.1 -%}
-            "code":"PLAC",
-            {% endif -%}
-            {% if EIP.2 -%}
-            "code":"FILL",
-            {% endif -%}
-            {% if EIP.1 -%}
-            "system":"http://terminology.hl7.org/CodeSystem/v2-0203",
-            {% endif -%}
-            {% if EIP.2 -%}
-            "system":"http://terminology.hl7.org/CodeSystem/v2-0203",
-            {% endif -%}
-        },
-    ],
-},
-
-{% endif -%}
\ No newline at end of file
+    "value":"{{ EIP.1.1 }}",
+    "value":"{{ EIP.2.1 }}",
```

Comment:
Please review this. Why do we have two value nodes here? EIP.1 and EIP.2 are different entities. Do we need an If condition here?

Reply:
The mapping spreadsheet `EIP[Identifier]` contains two 'identifier' properties, which points to `EI[Identifier]` template and will be unfolded to `value: ...`. The condition statements make little difference, there are same situations for the following `code` and `system` properties.

I'm not pretty sure how to update this template, do you have any suggestions?